src/SymbolPaths/include/SymbolPaths/QSettingsBasedStorageManager.h

```diff
@@ -12,7 +12,11 @@ namespace orbit_symbol_paths {
 class QSettingsBasedStorageManager : public PersistentStorageManager {
  public:
   void SavePaths(absl::Span<const std::filesystem::path> paths) override;
-  [[nodiscard]] virtual std::vector<std::filesystem::path> LoadPaths() override;
+  [[nodiscard]] std::vector<std::filesystem::path> LoadPaths() override;
```

Comment:
Double-check this line, LoadPaths is not changed in any other part.

Reply:
Actually, it might be a cause of compiling error?

Reply:
The change is only here, because its only removing the `virtual` keyword. It was something that I forgot to do in an older PR. Clang-tidy reminded me that the virtual keyword is redundant when there already is the keyword override
https://screenshot.googleplex.com/A9C2SE93eXgsDfX

Reply:
Thanks! :)